terraform/resources/langfuse.tf

```diff
@@ -0,0 +1,49 @@
+# S3 Bucket for current environment
+resource "aws_s3_bucket" "langfuse" {
+  bucket = "langfuse-${var.environment}-bucket"
+}
+
+# IAM user for Langfuse
+resource "aws_iam_user" "langfuse" {
+  name = "langfuse-${var.environment}-s3-user"
+}
+
+# Bucket policy
+resource "aws_iam_user_policy" "langfuse" {
+  name = "langfuse-${var.environment}-policy"
+  user = aws_iam_user.langfuse.name
+
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Effect = "Allow"
+        Action = [
+          "s3:PutObject",
+          "s3:GetObject",
+          "s3:DeleteObject",
+          "s3:ListBucket"
+        ]
+        Resource = [
+          aws_s3_bucket.langfuse.arn,
+          "${aws_s3_bucket.langfuse.arn}/*"
+        ]
+      }
+    ]
+  })
+}
+
+# Access key for the IAM user
+resource "aws_iam_access_key" "langfuse" {
+  user = aws_iam_user.langfuse.name
+}
+
+# Outputs
+output "langfuse_access_key_id" {
+  value = aws_iam_access_key.langfuse.id
+}
+
+output "langfuse_secret_key" {
+  value     = aws_iam_access_key.langfuse.secret
+  sensitive = true
+}
```

Comment:
If this works as expected, we should save this in a k8s secret through terraform and use [`secretKeyRef`](https://github.com/langfuse/langfuse-k8s/blob/689e3ef818dd8414d2e7a8f45948ad8698c85ac9/charts/langfuse/values.yaml#L514) to reference it in our values.yml